docs/news.rst

```diff
@@ -33,7 +33,8 @@ Backward-incompatible changes
     finalized and frozen. This change was needed to allow changing the settings
     in :meth:`scrapy.Spider.from_crawler`. If you want to access the final
     setting values in the spider code as early as possible you can do this in
-    :meth:`~scrapy.Spider.start_requests`. (:issue:`6038`)
+    :meth:`~scrapy.Spider.start_requests`. You can also use :signal:`~scrapy.signals.engine_started`
```

Comment:
Both `start_requests` and `engine_started` give access to fully initialized components. This just needs to say that from_crawler doesn't have it and that there are two options to get it.

Also, `:signal:` uses a signal name, not a fully qualified path to the object.